crates/sui-rpc-loadgen/src/payload/multi_get_transaction_blocks.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+
+use crate::payload::{MultiGetTransactionBlocks, ProcessPayload, RpcCommandProcessor, SignerInfo};
+use async_trait::async_trait;
+use futures::future::join_all;
+use tracing::log::warn;
+
+use super::validation::{check_transactions, chunk_entities};
+
+#[async_trait]
+impl<'a> ProcessPayload<'a, &'a MultiGetTransactionBlocks> for RpcCommandProcessor {
+    async fn process(
+        &'a self,
+        op: &'a MultiGetTransactionBlocks,
+        _signer_info: &Option<SignerInfo>,
+    ) -> Result<()> {
+        let clients = self.get_clients().await?;
+        let digests = &op.digests;
+
+        if op.digests.is_empty() {
+            warn!("No digests provided, skipping query");
+            return Ok(());
```

Comment:
prefer to panic here because this is not something the user would expect

Reply:
makes sense